pkg/accounting/rollup/rollup.go

```diff
@@ -54,30 +54,55 @@ func (r *Rollup) Run(ctx context.Context) (err error) {
 	}
 }
 
-// RollupRaws rolls up raw tally
-func (r *Rollup) RollupRaws(ctx context.Context) error {
+// Rollup aggregates storage and bandwidth amounts for the time interval
+func (r *Service) Rollup(ctx context.Context) error {
 	// only Rollup new things - get LastRollup
-	var latestTally time.Time
 	lastRollup, err := r.db.LastTimestamp(ctx, accounting.LastRollup)
+	r.logger.Info("last Rollup: " + lastRollup.String())
```

Comment:
log info after the err is checked

Reply:
oh shoot, that was just a debugging statement, removing.